core/lib/server/pub-capabilities.db.test.ts

```diff
@@ -0,0 +1,150 @@
+import { describe, expect, it } from "vitest";
```

Comment:
I pulled this into its own file because `pub.db.test.ts` has an issue of its transactions not working properly, I think because we aren't dynamically importing `seedCommunity`. this meant that I'd query for pubs only to get pubs created by other tests in the file. so this separates it out, but we should fix `pub.db.test.ts` so we can run its tests without the tests bumping into each other

Reply:
separating it seems good for organization anyways!